config_defaults.yml

```diff
@@ -1,29 +1,72 @@
 ---
 .travis.yml:
-  script: "\"bundle exec rake validate && bundle exec rake lint && bundle exec rake spec SPEC_OPTS='--format documentation'\""
+  supported_oss: &supported_oss
+    puppetlabs-acl:                      [windows]
+    puppetlabs-chocolatey:               [windows]
+    puppetlabs-dism:                     [windows]
+    puppetlabs-dsc:                      [windows]
+    puppetlabs-iis:                      [windows]
+    puppetlabs-powershell:               [windows]
+    puppetlabs-reboot:                   [windows]
+    puppetlabs-registry:                 [windows]
+    puppetlabs-sqlserver:                [windows]
+    puppetlabs-wsus_client:              [windows]
+    puppetlabs-inifile:                  [linux,windows]
+    puppetlabs-stdlib:                   [linux,windows]
```

Comment:
Eventually we'll be there with the other changes that have been suggested by @bmjen - for now there is a difference in the gemfile that gets extracted with the additional gems when Windows. However it could be the same.